src/Services/Logger.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Services;
+
+use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
+
+class Logger
```

Comment:
I was thinking this is actually a class decorator, meaning its main scope is still to log messages as classes implementing `Psr\Log\LoggerInterface;` would but we add more functionalities to it, like the possibility of logging according to a certain log level.

I'd maybe rename this class as:
```php
class LoggerDecorator implements LoggerInterface {
}
```
And use the same method signatures as on `LoggerInterface`.